### PR TITLE
[Ch13203] Intervention amendment notifications

### PR DESCRIPTION
**Pull request by csabadenes**
[Ch13203]

### CODE REVIEW

File: src/etools/applications/partners/views/interventions_v2.py [hunk 1]
```diff
@@ -298,12 +299,14 @@ def update(self, request, *args, **kwargs):
             nested_related_names=nested_related_names,
             **kwargs
         )
-
         validator = InterventionValid(instance, old=old_instance, user=request.user)
         if not validator.is_valid:
             logging.debug(validator.errors)
             raise ValidationError(validator.errors)
 
+        if old_instance and instance.in_amendment == False and old_instance.in_amendment == True:
```
Comment (domdinicola) [inline src/etools/applications/partners/views/interventions_v2.py]: you can use 
`not instance.in_amendment and instance.in_amendment`

Comment (csabadenes) [reply src/etools/applications/partners/views/interventions_v2.py]: I will change it now, thx

File: src/etools/applications/partners/views/interventions_v2.py [hunk 2]
```diff
@@ -298,12 +299,14 @@ def update(self, request, *args, **kwargs):
             nested_related_names=nested_related_names,
             **kwargs
         )
-
         validator = InterventionValid(instance, old=old_instance, user=request.user)
         if not validator.is_valid:
             logging.debug(validator.errors)
             raise ValidationError(validator.errors)
 
+        if old_instance and not instance.in_amendment and old_instance.in_amendment:
```
Comment (robertavram) [inline src/etools/applications/partners/views/interventions_v2.py]: Let's add a flag, feature-gating this bit.. we should be able to control whether in this country these kinds of emails get sent. Disable them temporarily. and they need to be disabled for now. you can use environment.switch

Comment (csabadenes) [reply src/etools/applications/partners/views/interventions_v2.py]: Ok. I will add the flag asap.

Comment (csabadenes) [reply src/etools/applications/partners/views/interventions_v2.py]: @robertavram i have added a tenantswitch check

**Pull request merged**